scripts/home/games/blit8/blit8.lua

```diff
@@ -0,0 +1,306 @@
+  elseif opcode1 == 1 then
+    PC = nnn - 2
+  elseif opcode1 == 2 then
+    stack[#stack + 1] = PC + 2
+    PC = nnn - 2
+  elseif opcode1 == 3 then
+    if V[x] == kk then
+      PC = PC + 2
+    end
+  elseif opcode1 == 4 then
+    if V[x] ~= kk then
+      PC = PC + 2
+    end
+  elseif opcode1 == 5 and opcode4 == 0 then
+    if V[x] == V[y] then
+      PC = PC + 2
+    end
+  elseif opcode1 == 6 then
+    V[x] = kk
+  elseif opcode1 == 7 then
+    V[x] = (V[x] + kk) % 256
+  elseif opcode1 == 8 then
+    if opcode4 == 0 then
+      V[x] = V[y]
+    elseif opcode4 == 1 then
+      V[x] = bit.bor(V[x], V[y])
+    elseif opcode4 == 2 then
+      V[x] = bit.band(V[x], V[y])
+    elseif opcode4 == 3 then
+      V[x] = bit.bxor(V[x], V[y])
+    elseif opcode4 == 4 then
+      V[x] = V[x] + V[y]
+      if V[x] > 255 then
+        V[x] = V[x] % 256
+        V[16] = 1
+      else
+        V[16] = 0
+      end
+    elseif opcode4 == 5 then
+      V[x] = V[x] - V[y]
+      if V[x] < 0 then
+        V[16] = 0
+        V[x] = V[x] % 256
+      else
+        V[16] = 1
+      end
+    elseif opcode4 == 6 then
+      V[16] = V[x] % 2
+      V[x] = math.floor(V[x] / 2)
+    elseif opcode4 == 7 then
+      V[x] = V[y] - V[x]
+      if V[x] < 0 then
+        V[16] = 0
+        V[x] = V[x] % 256
+      else
+        V[16] = 1
+      end
+    elseif opcode4 == 0xE then
+      V[16] = math.floor(V[x] / 128)
+      V[x] = (V[x] * 2) % 256
+    end
+  elseif opcode1 == 9 and opcode4 == 0 then
+    if V[x] ~= V[y] then
+      PC = PC + 2
+    end
+  elseif opcode1 == 0xA then
+    I = nnn
+  elseif opcode1 == 0xB then
+    PC = nnn + V[1] - 2
+  elseif opcode1 == 0xC then
+    V[x] = bit.band(math.floor(math.random() * 256), kk)
+  elseif opcode1 == 0xD then
+    local px, py = V[x] % 64, V[y] % 32
+    V[16] = 0
+    for j = 0, opcode4 - 1 do
+      if py + j > 31 then
+        break
+      end
+      local byte = RAM[I + j + 1]
+      for i = 0, 7 do
+        if px + i > 63 then
+          break
+        end
+        if bit.band(byte, 2 ^ (7 - i)) > 0 then
+          local index = (py + j) * 64 + px + i + 1
+          display[index] = not display[index]
+          if not display[index] then
+            V[16] = 1
+          end
+        end
+      end
+    end
+  elseif opcode1 == 0xE and opcode3 == 9 and opcode4 == 0xE then
+    if keys[V[x] + 1] then
+      PC = PC + 2
+    end
+  elseif opcode1 == 0xE and opcode3 == 0xA and opcode4 == 1 then
+    if not keys[V[x] + 1] then
+      PC = PC + 2
+    end
+  elseif opcode1 == 0xF then
+    if opcode3 == 0 and opcode4 == 7 then
+      V[x] = DT
+    elseif opcode3 == 0 and opcode4 == 0xA then
+      if not keypress then
+        PC = PC - 2
+      else
+        V[x] = keypress
+      end
+    elseif opcode3 == 1 and opcode4 == 5 then
+      DT = V[x]
+    elseif opcode3 == 1 and opcode4 == 8 then
+      ST = V[x]
+      if ST > 0 and commands then
+        commands.execAsync("/playsound note.pling @a ~ ~ ~")
```

Comment:
This should probably be replaced with Riko4's sound API.

Reply:
Thanks for spotting that! It was an oversight on my part. https://github.com/incinirate/Riko4/pull/32 should fix this.